cloud-core/src/main/java/cloud/commandframework/context/CommandContext.java

```diff
@@ -558,6 +559,24 @@ public void remove(final @NonNull CloudKey<?> key) {
         return this.getOptional(key).orElseGet(defaultSupplier);
     }
 
+    /**
+     * Get a value if it exists, else compute and store the value by the supplier and return it
+     *
+     * @param key             Argument key
+     * @param defaultSupplier Supplier of default value
+     * @param <T>             Argument type
```

Comment:
```suggestion
     * @param <T>             Value type
```

Reply:
This follows all other methods like it (eg: getOrSupplyDefault) so i'll leave it as-is